client/components/domains/transfer-domain-step/index.jsx

```diff
@@ -212,7 +212,7 @@ class TransferDomainStep extends React.Component {
 			);
 		} else if ( domainsWithPlansOnlyButNoPlan ) {
 			domainProductPriceText = translate(
-				'One additional year of domain registration included in paid plans.'
+				'One additional year of domain registration included in annual paid plans.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.